public/locales/en/translation.json

```
@@ -0,0 +1,28 @@
+{
+    "home": {
+        "title": "Federal centers of software development",
+        "welcome": "Welcome to the federal centers of software development",

```

AFAIK, this is how WSG calls them in English:

```suggestion
        "title": "National centers of software development",
        "welcome": "Welcome to the national centers of software development",
```